pom.xml

```diff
@@ -8,7 +8,28 @@
     <artifactId>SpringBootCalculator</artifactId>
     <version>1.0-SNAPSHOT</version>
 
- 
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>org.springframework.boot</groupId>
+                <artifactId>spring-boot-dependencies</artifactId>
+                <version>2.0.0.RELEASE</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-web</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-jooq</artifactId>
```

Comment:
at this moment the latest version is 3.10.6 https://mvnrepository.com/artifact/org.jooq/jooq/3.10.6
please check which version in `spring-boot-starter-jooq`